Siksha/Library/UIRepresentable/CalendarView.swift

```diff
@@ -146,8 +146,8 @@ class DateHeader: JTACMonthReusableView {
         
         leftButton.frame = CGRect(x: 0, y: 0, width: 10, height: 16)
         rightButton.frame = CGRect(x: 0, y: 0, width: 10, height: 16)
-        leftButton.setBackgroundImage(UIImage(named: "PrevDate"), for: .normal)
-        rightButton.setBackgroundImage(UIImage(named: "NextDate"), for: .normal)
+        leftButton.setBackgroundImage(UIImage(named: "PrevDate-orange"), for: .normal)
```

Comment:
```suggestion
        leftButton.setBackgroundImage(UIImage(named: "Prevdate-orange"), for: .normal)
```

오타때문에 왼쪽 화살표가 안보이네요
여기나 이미지 이름 바꿔야될 것 같아요

<img width="300" src="https://github.com/user-attachments/assets/52bfbe1b-8892-4e2d-960c-5825d0535219" />